common/lib/common-definitions/src/logger.ts

```diff
@@ -11,8 +11,17 @@ export type TelemetryEventCategory = "generic" | "error" | "performance";
 // General best practice is to explicitly log the fields you care about from objects
 export type TelemetryEventPropertyType = string | number | boolean | undefined;
 
+/**
+ * A property to be logged to telemetry containing both the value and a tag. Tags are generic strings that can be used
+ * to mark pieces of information that should be organized or handled differently by loggers in various first or third
+ * party scenarios. For example, tags are used to mark PII that should not be stored in logs.
+ */
+export interface ITaggedTelemetryPropertyType {
+    value: TelemetryEventPropertyType,
+    tag: string
```

Comment:
(nit) Curious why compiler does not bark at missing trailing comma, might be worth looking at it to ensure our code is uniform across different packages

Reply:
Changed but will leave this unresolved